### PR TITLE
Branch add sort property command

### LINKED ISSUE #164: Sorting property by price showing incorrect order
The current order of the property prices:
1. $1,000,000
2. $2,500,000
3. $800,000

image

### PR DESCRIPTION
**Pull request by Cp-John**

### CODE REVIEW

File: src/main/java/seedu/address/model/property/client/AskingPrice.java [hunk 1]
```diff
@@ -66,7 +66,9 @@ public boolean equals(Object other) {
 
     @Override
     public int compareTo(AskingPrice another) {
-        return this.askingPrice.compareTo(another.askingPrice);
+        int price1 = Integer.parseInt(this.askingPrice.substring(1).replace(",", ""));
+        int price2 = Integer.parseInt(another.askingPrice.substring(1).replace(",", ""));
```
Comment (w-yuchen) [inline src/main/java/seedu/address/model/property/client/AskingPrice.java]: i think according to the validation regex price can have cents. Should we still consider that for the price?

**Review by w-yuchen — COMMENTED** [review]
Looks good.

File: src/main/java/seedu/address/model/property/client/AskingPrice.java (hunk 1 above)
Comment (Cp-John) [reply src/main/java/seedu/address/model/property/client/AskingPrice.java]: fixed

**Review by dvdweien — APPROVED** [review]
LGTM

**Pull request merged**